### PR TITLE
Fixing TextDocumentSettingsIcon size #21645

### LINKED ISSUE #21645: Wrong size of TextDocumentSettingsIcon in @fluentui/react-icons-mdl2
### Environment Information

- **Browser and OS versions**: Chrome, Safari, Firefox (latest versions)

### Steps to reproduce:

  * Go to https://developer.microsoft.com/en-us/fluentui#/styles/web/icons#available-icons
  * Scroll down and click on **SVG Icons**
  * Scroll down and find icon _TextDocumentSettings_

#### Actual behavior:

  * The icon is not visible
  * The icon is very large after hovering over it

Screenshot 2022-02-04 at 11 45 38

#### Expected behavior:

 * The icon should look and behave in the same way as other icons

### Priorities and help requested:

Are you willing to submit a PR to fix? **Yes**

Requested priority: **Normal**

Products/sites affected: All products/sites using the [@fluentui/react-icons-mdl2](https://www.npmjs.com/package/@fluentui/react-icons-mdl2) package

#### Possible solution:

To remove `width` and `height` attributes from [TextDocumentSettingsIcon.tsx](https://github.com/microsoft/fluentui/blob/master/packages/react-icons-mdl2/src/components/TextDocumentSettingsIcon.tsx)
It is the only icon component having these properties. The component should look as follows:

```ts
import * as React from 'react';
import createSvgIcon from '../utils/createSvgIcon';

const TextDocumentSettingsIcon = createSvgIcon({
  svg: ({ classes }) => (
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 2048 2048">
      <path d="M1103 1920q23 37 52 68t62 60H128V0h1115l549 549v494q-63-22-128-29V640h-512V128H256v1792h847zm177-1701v293h293l-293-293zm-128 998q-13 15-25 30t-24 33H512v-128h640v65zm-640 319v-128h512v60q0 14-4 33t-6 35H512zm896-640v128H512V896h896zm512 704q0 31-6 61l124 51-49 119-124-52q-35 51-86 86l52 124-119 49-51-124q-30 6-61 6t-61-6l-51 124-119-49 52-124q-51-35-86-86l-124 52-49-119 124-51q-6-30-6-61t6-61l-124-51 49-119 124 52q18-25 39-47t47-39l-52-124 119-49 51 124q30-6 61-6t61 6l51-124 119 49-52 124q51 35 86 86l124-52 49 119-124 51q6 30 6 61zm-128 0q0-40-15-75t-41-61-61-41-75-15q-40 0-75 15t-61 41-41 61-15 75q0 40 15 75t41 61 61 41 75 15q40 0 75-15t61-41 41-61 15-75z" />
    </svg>
  ),
  displayName: 'TextDocumentSettingsIcon',
});

export default TextDocumentSettingsIcon;
```

### PR DESCRIPTION
**Pull request by marek-mihok**
Closes #21645

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/0f6b25f8b62395fab1f6648bedf1c2763ed2c50e'>0f6b25f8b62395fab1f6648bedf1c2763ed2c50e</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Avatar-mount_base.html">961</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Avatar-mount.html">887</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/BaseButton-mount_base.html">1012</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/BaseButton-mount.html">999</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Breadcrumb-mount_base.html">3304</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Breadcrumb-mount.html">3183</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/ButtonNext-mount_base.html">515</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/ButtonNext-mount.html">531</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Checkbox-mount_base.html">1704</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Checkbox-mount.html">1672</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/CheckboxBase-mount_base.html">1430</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/CheckboxBase-mount.html">1397</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/ChoiceGroup-mount_base.html">4955</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/ChoiceGroup-mount.html">5120</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/ComboBox-mount_base.html">1019</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/ComboBox-mount.html">1057</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/CommandBar-mount_base.html">11698</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/CommandBar-mount.html">11400</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/ContextualMenu-mount_base.html">9510</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/ContextualMenu-mount.html">9568</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/DefaultButton-mount_base.html">1237</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/DefaultButton-mount.html">1217</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/DetailsRow-mount_base.html">4015</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/DetailsRow-mount.html">3991</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/DetailsRowFast-mount_base.html">4180</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/DetailsRowFast-mount.html">3948</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/DetailsRowNoStyles-mount_base.html">3770</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/DetailsRowNoStyles-mount.html">3867</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Dialog-mount_base.html">3207</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Dialog-mount.html">3220</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/DocumentCardTitle-mount_base.html">206</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/DocumentCardTitle-mount.html">196</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Dropdown-mount_base.html">3518</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Dropdown-mount.html">3532</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/FluentProviderNext-mount_base.html">2085</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/FluentProviderNext-mount.html">2046</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/FluentProviderWithTheme-mount_base.html">215</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/FluentProviderWithTheme-mount.html">202</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">136</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/FluentProviderWithTheme-virtual-rerender.html">143</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">238</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">266</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/FocusTrapZone-mount_base.html">1955</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/FocusTrapZone-mount.html">1978</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/FocusZone-mount_base.html">1878</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/FocusZone-mount.html">1984</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/IconButton-mount_base.html">1899</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/IconButton-mount.html">1902</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Label-mount_base.html">388</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Label-mount.html">419</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Layer-mount_base.html">3199</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Layer-mount.html">3178</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Link-mount_base.html">529</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Link-mount.html">544</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/MakeStyles-mount_base.html">1792</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/MakeStyles-mount.html">1806</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/MenuButton-mount_base.html">1600</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/MenuButton-mount.html">1632</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/MessageBar-mount_base.html">2261</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/MessageBar-mount.html">2200</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Nav-mount_base.html">3604</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Nav-mount.html">3611</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/OverflowSet-mount_base.html">1206</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/OverflowSet-mount.html">1207</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Panel-mount_base.html">2730</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Panel-mount.html">2721</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Persona-mount_base.html">948</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Persona-mount.html">927</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Pivot-mount_base.html">1577</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Pivot-mount.html">1597</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/PrimaryButton-mount_base.html">1404</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/PrimaryButton-mount.html">1446</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Rating-mount_base.html">8213</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Rating-mount.html">8213</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/SearchBox-mount_base.html">1496</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/SearchBox-mount.html">1463</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Shimmer-mount_base.html">2692</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Shimmer-mount.html">2757</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Slider-mount_base.html">2149</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Slider-mount.html">2128</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/SpinButton-mount_base.html">5414</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/SpinButton-mount.html">5386</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Spinner-mount_base.html">482</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Spinner-mount.html">482</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/SplitButton-mount_base.html">3413</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/SplitButton-mount.html">3512</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Stack-mount_base.html">586</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Stack-mount.html">548</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/StackWithIntrinsicChildren-mount_base.html">2433</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/StackWithIntrinsicChildren-mount.html">2465</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/StackWithTextChildren-mount_base.html">5491</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/StackWithTextChildren-mount.html">5579</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/SwatchColorPicker-mount_base.html">12078</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/SwatchColorPicker-mount.html">12190</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/TagPicker-mount_base.html">2870</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/TagPicker-mount.html">2902</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/TeachingBubble-mount_base.html">14249</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/TeachingBubble-mount.html">14198</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Text-mount_base.html">477</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Text-mount.html">482</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/TextField-mount_base.html">1521</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/TextField-mount.html">1532</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/ThemeProvider-mount_base.html">1320</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/ThemeProvider-mount.html">1337</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/ThemeProvider-virtual-rerender_base.html">681</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/ThemeProvider-virtual-rerender.html">688</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1956</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2050</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Toggle-mount_base.html">881</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/Toggle-mount.html">886</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/buttonNative-mount_base.html">169</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21646/perf-test/buttonNative-mount.html">162</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

File: change/@fluentui-react-icons-mdl2-cde5eb5f-56eb-49e1-a5e4-c28fe64e0c63.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-icons-mdl2-cde5eb5f-56eb-49e1-a5e4-c28fe64e0c63.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=21646&commitId=0d34d8b05562b616993eabc67312f1726e6dd23b&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Review by tomi-msft — APPROVED** [review]
LGTM!

File: change/@fluentui-react-icons-mdl2-cde5eb5f-56eb-49e1-a5e4-c28fe64e0c63.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-react-icons-mdl2-cde5eb5f-56eb-49e1-a5e4-c28fe64e0c63.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=21646&commitId=7cb31495021eaf056dadd5c76183867a71a7be38&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

Comment (behowell) [reply change/@fluentui-react-icons-mdl2-cde5eb5f-56eb-49e1-a5e4-c28fe64e0c63.json:1]: This is a flaky VR test unrelated to this PR.

**Review by behowell — APPROVED** [review]

**Pull request merged**